.travis.yml

```diff
@@ -1,6 +1,24 @@
+sudo: false
+
 language: groovy
+
 jdk:
-- oraclejdk8
+  - oraclejdk7
+  - oraclejdk8
+
+env:
+  - GRADLE_VERSION=1.12
+  - GRADLE_VERSION=2.4
+  - GRADLE_VERSION=2.5
+
+before_script:
+  - cp -rf gradle/wrapper/gradle-wrapper.template gradle/wrapper/gradle-wrapper.properties
```

Comment:
My comment could be confusing.

By default in `gradle-wrapper.properties` there should be:
> distributionUrl=http\://services.gradle.org/distributions/gradle-1.12-bin.zip

To test it with 2.x you could append the following line to that file:
> echo "distributionUrl=http\://services.gradle.org/distributions/gradle-2.5-bin.zip" >> gradle/wrapper/gradle-wrapper.properties

To make it run with Gradle 2.5.

See https://github.com/4finance/micro-infra-spring/blob/master/gradle/runBootMicroserviceAcceptanceTests.sh#L10